### PR TITLE
Add contributor documentation

### PR DESCRIPTION
**Pull request by dominikschulz**
Inspiration from https://manishearth.github.io/blog/2016/01/03/making-your-open-source-project-newcomer-friendly/
and https://manishearth.github.io/blog/2016/01/03/making-your-open-source-project-newcomer-friendly/

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+```
+
+## Pull Request Checklist
+
+* Use that [latest stable Go release](https://golang.org/dl/)
+
+* Branch from master and, if needed, rebase to the current master branch before submitting your pull request.
+  If it doesn't merge cleanly with master you will be asked to rebase your changes.
+
+* Commits should be as small as possible, while ensuring that each commit is corrent indepently.
```
Comment (dewey) [inline CONTRIBUTING.md]: Typo, `independently`. Don't really understand what's meant by corrent. Should probably be rephrased.

Comment (dominikschulz) [inline CONTRIBUTING.md]: Yes, indeed. Will do.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+Should you wish to work on an issue, please claim it first by commenting on the GitHub issue you want to work on it.
+This will prevent duplicated efforts from contributors.
+
+Please check the [`help-wanted`](https://github.com/justwatchcom/gopass/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) label to find issues that need help.
+If you have questions about one of the issues please comment on them and one of the maintainers
+will try to clarify it.
+
+For quickly compiling and testing your changes do:
+```
+go build
```
Comment (metalmatze) [inline CONTRIBUTING.md]: Why not let them run `make build`? Simply running `go build` will not include any tags etc.
Because it's more simple?

Comment (dominikschulz) [inline CONTRIBUTING.md]: If you run `make build` it won't create an executable name `gopass` but something like `gopass-${GOOS}-${GOARCH}`. I think `go build` is simpler.

Keep in mind this is meant for people wanting to work on gopass, not for regular users.

**Pull request merged**